### PR TITLE
EZP-29858: Fix broken Core tests

### PR DESCRIPTION
**Pull request by mnocon**
| Question           | Answer
| ------------------ | ------------------
| **JIRA issue**     | Follow up to https://jira.ez.no/browse/EZP-29858
| **Bug/Improvement**| no
| **New feature**    | no
| **Target version** | master (3.0)
| **BC breaks**      | no
| **Tests pass**     | yes
| **Doc needed**     | no

In https://jira.ez.no/browse/EZP-29858 I've been refactoring BehatBundle (and it's usage in Kernel) and missed that changes are required to these tests (making them broken in the process).

Failing on master: https://travis-ci.org/ezsystems/ezplatform/jobs/533048313
Passing on 2.5: https://travis-ci.org/ezsystems/ezplatform/jobs/532772906

This PR contains two changes:
1) Adds execution of Core Kernel suite to Travis matrix - up to discussion, it's currently executed on `ezplatform` repository. Do we want to include it here/backport it to 6.7? If not this can be treated as a TMP commit that proves that the tests are passing again
2) Fix tests so that they're passing - the behaviour in tests is exactly the same, only different step is used.

**TODO**:
- [x] Implement feature / fix a bug.
- [x] Implement tests.
- [x] Fix new code according to Coding Standards (`$ composer fix-cs`).
- [x] Ask for Code Review.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -36,6 +36,12 @@ matrix:
       env: TEST_CONFIG="phpunit.xml"
     - php: 7.1
       env: REST_TEST_CONFIG="phpunit-functional-rest.xml" SYMFONY_ENV=behat SYMFONY_DEBUG=1 SF_CMD="ez:behat:create-language 'pol-PL' 'Polish (polski)'"
+    - name: "Kernel Core tests"
```
Comment (alongosz) [inline .travis.yml]: Please say here that it refers to the behat test, from the perspective of Travis jobs list it's not obvious.

**Comment by mnocon** [comment]
> Do they provide a lot of value? Does back-porting it require extra changes?

No changes needed, as the tests are currently running and passing on ezplatform's Travis 1.7, 1.13 and 2.4, 2.5 (and with this PR: master).

About the value - I'm not sure. There are only 7 scenarios.
- 2 of them are about siteaccess aware Symfony commands
- 4 are about viewing Content on the frontend: that it's possible to preview an item and that UnauthorizedExceptions are handled as they should be.
- 1 tests QueryController

For me it makes sense to run tests that are stored in Kernel on Kernel PRs, but given that they're already run in ezplatform repo I'd be ok with skipping it as well.

@alongosz I'll let you decide (if we keep it then I'll correct the job name and backport first commit to 6.7, else I'll remove it)

**Comment by alongosz** [comment]
@mnocon from my POV we can skip back-porting, some of these tests are covered by integration anyway.

**Comment by mnocon** [comment]
@alongosz removed the Travis job commit, now there are test changes only

**Comment by mnocon** [comment]
Aaaand added back again 🙃 Let's wait for Travis to agree

**Comment by mnocon** [comment]
@lserwatka this can be merged

**Pull request merged**